### ISSUE TITLE
Allow access to create app action from anywhere & everywhere

### ISSUE THREAD

**Issue by serenamarie125** [openshiftio/openshift.io#4190]
As part of the design thinking session around _Goal A - Improve Onboarding for novice_, **always allow access to the create application action** was prioritized as an issue to address.

**Comment by sdash-redhat** [comment]
Some explorations/WF are here 
**https://redhat.invisionapp.com/share/TDNQ0T2BH3K**

There are 2 approaches, or we can think of combining them. 
Please share your feedback and thoughts.

**Comment by serenamarie125** [comment]
@sdash-redhat has this Invision been updated to reflect input?

**Comment by sdash-redhat** [comment]
Here it is as Invisio link: 
https://redhat.invisionapp.com/share/HJNWRKFMY6U#/318416629_-4190_-_Title
@serenamarie125

**Comment by joshuawilson** [comment]
I don't think this has been done by eng, nor has an issue been created for eng to track it. 

@serenamarie125 I'm going to reopen it until we get it moved over.

**Comment by serenamarie125** [comment]
@sdash-redhat please put this on the December 19th stakeholder agenda

**Comment by sdash-redhat** [comment]
Based on today's stakeholders' review call, we will iterate on this design: 
1. "Find a Space to Join" should change to "Search Spaces". Also related titles, placeholder texts must change. 
2. "Join" related action triggers should be removed. 
3. type ahead will be still parked for future. For the time being the placeholder search must provide hint that the user must hit Enter to activate search. 

Moving the status to in-progress.

**Comment by sdash-redhat** [comment]
Based on today's team discussion, we can relook at it once Ashish's sketch is available with the new IA
